### PR TITLE
Features/#4 status2019 chp

### PR DESCRIPTION
**Pull request by ClaraBuettner**
Deals with #4 
It includes #21 and should be merged afterwards.

This branch adds existing CHP plans for the status 2019 scenario. I tested it for Schleswig-Holstein.

### CODE REVIEW

File: src/egon/data/datasets/chp/__init__.py [hunk 1]
```diff
@@ -310,6 +314,125 @@ def insert_biomass_chp(scenario):
...
+                "Steinkohle",
+            ]
+        )
+    ]
+
+    mastr.Inbetriebnahmedatum = pd.to_datetime(mastr.Inbetriebnahmedatum)
+    mastr.DatumEndgueltigeStilllegung = pd.to_datetime(
+        mastr.DatumEndgueltigeStilllegung
+    )
+    mastr = mastr.loc[mastr.Inbetriebnahmedatum <= "2019-12-31"]
```
Comment (ulfmueller) [inline src/egon/data/datasets/chp/__init__.py]: maybe it makes sense to refer to this [value](https://github.com/openego/powerd-data/blob/dev/src/egon/data/datasets.yml#L309) in the datasets.yaml just to keep track and know where we filter by this - might be useful for the update functionality...

File: src/egon/data/datasets/chp/__init__.py [hunk 2]
```diff
@@ -310,6 +314,125 @@ def insert_biomass_chp(scenario):
...
+
+    mastr = mastr.loc[
+        (mastr.DatumEndgueltigeStilllegung >= "2019-12-31")
```
Comment (ulfmueller) [inline src/egon/data/datasets/chp/__init__.py]: see comment above

**Review by ulfmueller — APPROVED** [review]
looks good. To address my comment would be nice but not so important...

File: src/egon/data/datasets/chp/__init__.py (hunk 1 above)
Comment (ClaraBuettner) [reply src/egon/data/datasets/chp/__init__.py]: Done in [c4dfc62](https://github.com/openego/powerd-data/pull/24/commits/c4dfc62e088ec41fdf97ed2b0cb3a8c4cd7301d4)

File: src/egon/data/datasets/chp/__init__.py (hunk 2 above)
Comment (ClaraBuettner) [reply src/egon/data/datasets/chp/__init__.py]: Done in [c4dfc62](https://github.com/openego/powerd-data/pull/24/commits/c4dfc62e088ec41fdf97ed2b0cb3a8c4cd7301d4)

**Pull request merged**